src/components/Grid/Grid.spec.tsx

```diff
@@ -0,0 +1,301 @@
+  return Array(amount)
+    .fill(null)
+    .map((item, index) => {
+      return {
+        item: <Card stacked info={<div>info</div>} />,
+        key: withKey ? `key-${index}` : undefined,
+      };
+    });
+}
+
+describe('Grid', () => {
+  const createDriver = createUniDriverFactory(gridDriverFactory);
+
+  it('should render', async () => {
+    const driver = createDriver(<Grid />);
+    expect(await driver.exists()).toBe(true);
+  });
+
+  it('should calculate max items per row', () => {
+    spyOn(GridUtils, 'itemsPerRowWidth');
+    const expectedListWidth = 520;
+    const expectedMaxItemsPerRow = 5;
+    const expectedMinItemWidth = 120;
+    const expectedColumnGap = 10;
+
+    createDriver(
+      <Grid
+        width={expectedListWidth}
+        maxColumns={expectedMaxItemsPerRow}
+        minColumnWidth={expectedMinItemWidth}
+        columnGap={expectedColumnGap}
+        items={generateItems()}
+      />,
+    );
+
+    expect(GridUtils.itemsPerRowWidth).toHaveBeenCalledWith(
+      expectedListWidth,
+      expectedMinItemWidth,
+      expectedMaxItemsPerRow,
+      expectedColumnGap,
+    );
+  });
+
+  it('should accept dividerWidth as number', async () => {
+    const dividerWidth = 3;
+    const expectedDividerWidth = `${dividerWidth}px`;
+    const driver = createDriver(
+      <Grid
+        dividerWidth={dividerWidth}
+        width={1}
+        withDivider
+        items={generateItems()}
+      />,
+    );
+
+    expect(await driver.isDividerWidth(expectedDividerWidth)).toEqual(true);
+  });
+
+  it('should accept dividerWidth as string', async () => {
+    const expectedDividerWidth = '7px';
+    const driver = createDriver(
+      <Grid
+        dividerWidth={expectedDividerWidth}
+        width={1}
+        withDivider
+        items={generateItems()}
+      />,
+    );
+
+    expect(await driver.isDividerWidth(expectedDividerWidth)).toEqual(true);
+  });
+
+  it('should limit items per row depending on the width of the List', async () => {
+    const expectedItemsPerRow = 4;
+    spyOn(GridUtils, 'itemsPerRowWidth').and.returnValue(expectedItemsPerRow);
+
+    const driver = createDriver(<Grid width={1} />);
+
+    expect(await driver.isItemsPerRow(expectedItemsPerRow)).toBe(true);
+  });
+
+  it('should use amount of items as maxColumns if lower then maxColumns', async () => {
+    const expectedItemsPerRow = 4;
+
+    const driver = createDriver(
+      <Grid
+        items={generateItems(expectedItemsPerRow)}
+        maxColumns={expectedItemsPerRow + 1}
+        width={1000}
+      />,
+    );
+
+    expect(await driver.isItemsPerRow(expectedItemsPerRow)).toBe(true);
+  });
+
+  it('should use 1 default amount of items ', async () => {
+    const driver = createDriver(<Grid items={generateItems()} width={1} />);
+
+    expect(await driver.isItemsPerRow(1)).toBe(true);
+  });
+
+  it('should use default max item width', async () => {
+    const expectedItemMaxWidth = '100vw';
```

Comment:
100vw of the iframe or the window getting the same result because we still limited by the container
but after checking the result of 100vw it is taking as much space that it can instead of what it need. so changing to 100%

Reply:
We checked, it, 100% does not provide the desired behavior, as it will not really stretch to consume all the space available, as it is just 100% of the container which will not necessarily get the full width of the window/iframe, so we will leave it as 100vw